src/openrct2/ride/Vehicle.h

```diff
@@ -373,6 +373,8 @@ struct Vehicle : EntityBase
     void UpdateLandscapeDoor() const;
     void UpdateLandscapeDoorBackwards() const;
     int32_t CalculateRiderBraking() const;
+
+    void Sub6DCE02(const Ride& curRide);
```

Comment:
Please rename this `Loc6DCE02` as I requested, not `Sub6DCE02`. It wasn’t a subroutine, otherwise it wouldn’t have been a goto in the first place.